docs/comparison.rst

```diff
@@ -0,0 +1,83 @@
+
+As such, it has relatively poor performance compared with several other simulators. Deciding which tool to use will ultimately depend on what factors are important to the user. We recommend pyuvsim as a reference tool to verify the results of other simulators, and for producing limited datasets for comparison with observations (particularly on long baselines). Users will need access to a lot of memory and a large number of processor cores to take advantage of pyuvsim's accuracy and precision, ideally on a dedicated high-performance computing cluster.
+
+On this page, we compare pyuvsim's features and performance to several other visibility simulators.
+
+Precision Radio Interferometry Simulator (PRISim)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A python package for generating full-sky simulations.
+
+Advantages:
+
+1. Faster due to vectorized calculations. 
+2. Well-tested through years of use.
+3. Support for a range of sky models including point sources, diffuse emission, and spectral cubes.
+4. Support for tracking as well as transit telescopes.
+5. Parallelized over baselines, frequencies, or sky model/catalog.
+6. Has been tested using imaging with CASA.
+
+Disadvantages:
+
+1. MPI overhead is large if time axis in simulation is long.
+2. Supports only one polarization at a time.
+3. Does not support non-identical antenna patterns.
+
+Fast Holographic Deconvolution (FHD)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+FHD is a data calibration and modeling framework written in IDL by researchers at the University of Washington for analyzing data from the Murchison Widefield Array (MWA) in Western Australia. It calculates sky models by convolving a high-resolution psf with the exact DFT of point sources. This is approach is more efficient when working with wide primary beams, which will be narrower in Fourier space and thus limit the size of the convolution to be performed.
+
+Advantages:
+
+1. Fast.
+2. Well-tested through years of use.
+3. Support for diffuse foreground models.
+4. Support for discrete pointings.
+5. Uses IDL's native parallelization efficiently.
+
+Disadvantages:
+
+1. Written in the proprietary IDL language.
+2. Designed for the MWA, and has limited support for other instruments.
+3. The discrete convolution in Fourier Space can introduce aliasing and ringing artifacts at a level relevant to 21cm cosmology.
+4. Measurement equation formalism relies on the flat-sky approximation, limiting curved sky effects.
+5. Primary beam motion on the sky is limited to the "snapshot" size, not the time step size.
+6. Not parallelized over frequency.
```

Comment:
This last point is a bit confusing. It's true that visibilities are calculated separately by frequency, but they all use a single UV plane, so there's only one DFT of all the sources.

Reply:
Yeah, 6 is nuanced. Part of the speed is that it does not redo uv plane calculations of smooth spectrum sources. Is this infinitely parallel (did all the calculations at once!) or not parallel at all?

Reply:
For 6, I was thinking of the loop over frequency done by `vis_simulate.pro` if `model_uvf_arr` has a frequency axis. This is the case for simulating EoR-like signals, and has been a big slowdown for my work with FHD in the past. I'm not sure how much support there is for signal spectral modeling (vs. foregrounds).

Reply:
Yeah, this is why it is subtle. FHD is very slow for EoR signals, but it was designed to simulate smooth spectrum foregrounds and at that task it is fast.

Just for dissimenating information, in typical useage all the foregrounds are assumed to have the same spectral index over the simulated band. But the design allows for the foregrounds to be Taylor expanded in spectral shape, so sources with different spectral indices have a different linear term in the Taylor expansion and sources with curvature pick up higher terms. The simulation speed scales with the number of terms in the Taylor expansion. So even if you have thousands of frequency channels, a third order spectral foreground model is only three times slower than a single frequency channel (holding visibility # constant). 

This of course falls apart for the EoR, which is the worst case scenario for this kind of simulation approach. In this limit it falls back to the same approach as pyuvsim (different model for each frequency).

Reply:
Would this be a fair summary of this discussion, for inclusion on the list?

>FHD is optimized for smooth-spectrum sources but is not designed to handle complex spectral structure. 

I can probably split this sentence between "advantages" and "disadvantages".